src/Api/reviewPoster.ts

```diff
@@ -1,9 +1,15 @@
+import { AxiosResponse } from 'axios';
+
 import api from './api';
 
 const SPECIFIED_CATEGORY_ID = '63bd141d93836272216d324a';
 
-export const getSpecifiedReviewPoster = () =>
-  api.get(`/posts/channel/${SPECIFIED_CATEGORY_ID}`);
+// export const getSpecifiedReviewPoster = () =>
+//   api.get(`/posts/channel/${SPECIFIED_CATEGORY_ID}`);
 
 export const getAllReviewPoster = (channelId: string) =>
   api.get(`/posts/channel/${channelId}`);
+
+export const getSpecifiedReviewPoster = <T extends unknown[]>(): Promise<
+  AxiosResponse<T>
+> => api.get<T>(`/posts/channel/${SPECIFIED_CATEGORY_ID}`);
```

Comment:
p3
- 음.. 애초에 getSpecifiedReviewPoster에서 받을 데이터를 타입으로 명시하는게 좀더 좋겠네요!
- Axios의 나머지 데이터를 굳이 view레벨에서 알 필요가 있을까 싶어서, data를 추출한 값을 반환하는게 view에서 처리하는게 좀더 편리할 것 같다는 생각이 드네요

```suggestion
export const getSpecifiedReviewPoster = async (): Promise<ReviewPosterType[]> => {
  const { data } = await api.get<ReviewPosterType[]>(`/posts/channel/${SPECIFIED_CATEGORY_ID}`);
  return data;
};
```

Reply:
```js
category: extractCategoryCondition(categoryResponse as Category[]),
```

`as`로 처리된 부분도 as없이 해소될 것 같아요

Reply:
감사합니다. `as` 강제로 타입 지정해서 고민했는데 view 레벨 전에 추출하니 편하네요. 👍